### PR TITLE
Add shopping cart integration

### PR DESCRIPTION
**Pull request by YaroslavYarmoshyk**

### CODE REVIEW

File: src/main/java/com/online/bookstore/model/CartItem.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+public class CartItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ToString.Exclude
+    @ManyToOne
+    @JoinColumn(name = "shopping_cart_id")
+    private ShoppingCart shoppingCart;
+    @ToString.Exclude
+    @OneToOne
```
Comment (Vasul-Pankiv) [inline src/main/java/com/online/bookstore/model/CartItem.java]:
```suggestion
    @ManyToOne
```

File: src/main/java/com/online/bookstore/mapper/CartItemMapper.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import com.online.bookstore.model.ShoppingCart;
+import java.util.Set;
+import org.mapstruct.AfterMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+
+@Mapper(config = MapperConfig.class, uses = BookMapper.class)
+public interface CartItemMapper {
+
```
Comment (Vasul-Pankiv) [inline src/main/java/com/online/bookstore/mapper/CartItemMapper.java]: Do not start class with empty line

Comment (Vasul-Pankiv) [reply src/main/java/com/online/bookstore/mapper/CartItemMapper.java]: fix it everywhere

**Review by Vasul-Pankiv — CHANGES_REQUESTED** [review]
Great Job

**Review by koval-kn — APPROVED** [review]

**Pull request merged**